plugins/modules/ndo_l3out_node_routing_policy.py

```diff
@@ -0,0 +1,515 @@
+                                path="{0}/bgpTimerPol/keepAliveInterval".format(update_path),
+                                value=bgp_node_settings.get("keep_alive_interval"),
+                            )
+                        )
+
+                    if bgp_node_settings.get("hold_interval") is not None and proposed_payload.get("bgpTimerPol").get("holdInterval") != bgp_node_settings.get(
+                        "hold_interval"
+                    ):
+                        proposed_payload["bgpTimerPol"]["holdInterval"] = bgp_node_settings.get("hold_interval")
+                        ops.append(
+                            dict(
+                                op="replace",
+                                path="{0}/bgpTimerPol/holdInterval".format(update_path),
+                                value=bgp_node_settings.get("hold_interval"),
+                            )
+                        )
+
+                    if bgp_node_settings.get("stale_interval") is not None and proposed_payload.get("bgpTimerPol").get(
+                        "staleInterval"
+                    ) != bgp_node_settings.get("stale_interval"):
+                        proposed_payload["bgpTimerPol"]["staleInterval"] = bgp_node_settings.get("stale_interval")
+                        ops.append(
+                            dict(
+                                op="replace",
+                                path="{0}/bgpTimerPol/staleInterval".format(update_path),
+                                value=bgp_node_settings.get("stale_interval"),
+                            )
+                        )
+
+                    if bgp_node_settings.get("max_as_limit") is not None and proposed_payload.get("bgpTimerPol").get("maxAslimit") != bgp_node_settings.get(
+                        "max_as_limit"
+                    ):
+                        proposed_payload["bgpTimerPol"]["maxAslimit"] = bgp_node_settings.get("max_as_limit")
+                        ops.append(
+                            dict(
+                                op="replace",
+                                path="{0}/bgpTimerPol/maxAslimit".format(update_path),
+                                value=bgp_node_settings.get("max_as_limit"),
+                            )
+                        )
+
+            mso.sanitize(proposed_payload)
+        else:
+            if not (bfd_multi_hop_settings or bgp_node_settings or as_path_multipath_relax):
+                mso.fail_json(
+                    msg="At least one of the following attributes must be specified when creating L3Out Node Routing Policy:"
+                    + " 'bfd_multi_hop_settings', 'bgp_node_settings', or 'as_path_multipath_relax'."
+                )
+
+            payload = dict(name=name)
+            if description:
+                payload["description"] = description
+
+            if as_path_multipath_relax:  # initially ignores None and "" (empty string)
+                # enabled = true
```

Comment:
Are the comments required?

Reply:
Yes, we are translating the value from string to bool and the empty string to make the removal decision. So it will be helpful for future reference.